### DISCUSSION TITLE
Is there a way to change default KDF iterations for new users?

### DISCUSSION THREAD

**Discussion by ronnie-morgan** [dani-garcia/vaultwarden#3140] [Q&A]
I changed password iterations in Admin/General Setup, but any new users still have 100000 for KDF iterations.  And so far, I can find no way to make a different count the minimum for my server.  Is this a feature request, or am I doing something wrong?

**Comment by BlackDex** [comment]
That setting is only for the server side where the hash sent by the clients is stored. That setting is not used by the clients.

I'm not sure from the top of my head of ours even possible.

**Comment by NeoLizzard** [comment]
I made a suggestion here about how it may be perhaps possible:
https://github.com/dani-garcia/vaultwarden/discussions/3162

**Reply by BlackDex** [reply]
What you did there has nothing to do with the client-side iteration, that is only for storing the password hash by Vaultwarden.
For which i also just created a PR #3163, which will update the server-side to at least 350_000 iterations instead of 100_000. But it now also will update the current stored value if the iterations are changed globally.

For the client-side you still need to change it manually. That is not something we can do on the server side, since it requires the actual master-password, which the server never receives.
